packages/client/hmi-client/src/components/operator/tera-operator-menu.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<section ref="operatorMenu" :style="style">
+		<Button
+			v-if="node"
+			ref="openMenuButton"
+			class="button"
+			type="button"
+			size="small"
+			@click="showMenu"
+			icon="pi pi-plus"
+			:style="{
+				top: `${currentPortPosition.y - 18}px`,
+				left: `${currentPortPosition.x + 40}px`
+			}"
+			aria-haspopup="true"
+			aria-controls="overlay_menu"
+			severity="secondary"
+		/>
+		<Menu ref="menu" id="overlay_menu" :model="menuItems" :popup="true" />
```

Comment:
This should display the main menu items from the Workflow but filtered by types.
We can have a chat about it if you want more details.

Reply:
Ya, that is the plan with the next PR. 
Was just looking into the `WorkflowRegistry()` in `tera-workflow.vue` any suggestions are welcome :)